app/assets/javascripts/admin/models/web-hook.js.es6

```diff
@@ -22,8 +24,18 @@ export default RestModel.extend({
   },
 
   @observes('category_ids')
-  updateCategoryFilters() {
-    this.set('categoryFilters', Category.findByIds(this.get('category_ids')));
+  updateCategoriesFilter() {
+    this.set('categoriesFilter', Category.findByIds(this.get('category_ids')));
+  },
+
+  @observes('group_ids')
+  updateGroupsFilter() {
+    const groupIds = this.get('group_ids');
+    this.set('groupsFilterInName', Discourse.Site.currentProp('groups').filter(g => groupIds.includes(g.id)).map(g => g.name));
```

Comment:
`filter` and `map` can probably be reduced to just a single loop with a condition